cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/r4/R4CareGapStatusEvaluator.java

```diff
@@ -89,12 +99,39 @@ private CareGapsStatusCode getGapStatus(Measure measure, MeasureReportGroupCompo
 
         if (Boolean.TRUE.equals(inDenominator.getValue())
                 && ((isPositive && !inNumerator.getValue()) || (!isPositive && inNumerator.getValue()))) {
-            return CareGapsStatusCode.OPEN_GAP;
+            return getOpenOrProspectiveStatus(measureReportGroup, reportDate);
         }
 
         return CareGapsStatusCode.CLOSED_GAP;
     }
 
+    private CareGapsStatusCode getOpenOrProspectiveStatus(
```

Comment:
Might be a good idea to add a comment about the algorithm

Reply:
this found on the javaDoc on the main class method